worlds/cvcotm/options.py

```diff
@@ -0,0 +1,160 @@
+from dataclasses import dataclass
+from Options import Choice, Range, Toggle, PerGameCommonOptions, StartInventoryPool, DefaultOnToggle
+
+
+class IgnoreCleansing(Toggle):
+    """Makes the Cleansing not logically expected to traverse the water in Underground Waterway."""
+    display_name = "Ignore Cleansing"
+
+
+class AutoRun(Toggle):
+    """Causes Nathan to always run when pressing left or right without having to double-tap."""
+    display_name = "Auto Run"
+
+
+class DSSPatch(Toggle):
+    """Patches being able to pause during the DSS startup animation and switch the cards in the menu to use any combos
+    you don't currently have."""
+    display_name = "DSS Patch"
+
+
+class AlwaysAllowSpeedDash(Toggle):
+    """Allows activating the speed dash combo without needing the Pluto and Griffin cards first."""
+    display_name = "Always Allow Speed Dash"
+
+
+class BreakIronMaidens(Toggle):
+    """Breaks the iron maiden barriers from the start of the game, so you will not have to climb Chapel Tower and defeat
+    Adramelech to go past them."""
+    display_name = "Break Iron Maidens"
+
+
+class RequiredLastKeys(Range):
+    """How many Last Keys are needed to open the door to the Ceremonial
+    Room. This will lower if higher than Available Last Keys."""
```

Comment:
If you haven't already, I recommend reviewing how the options look on the webhost to see if it looks good. I believe it respects your linebreaks currently, and sometimes that leads to situations like this one:
![313364842-af77e75c-29fe-4afe-b3aa-85d8d960717c](https://github.com/ArchipelagoMW/Archipelago/assets/24858499/5d339297-4002-4fb7-b8b7-a7967796effe)

Reply:
I have been testing the options summary on the WebHost and I can say that they look good to me.